### PR TITLE
[native] Remove unnecessary includes from PrestoServer.cpp

### PR DESCRIPTION
**Pull request by amitkdutta**

### CODE REVIEW

**Review by gggrace14 — APPROVED** [review]
To make sure everything compiles okay, better to start the local server once.

**Comment by amitkdutta** [comment]
> To make sure everything compiles okay, better to start the local server once.

Yes, started server locally.

File: presto-native-execution/presto_cpp/main/PrestoServer.h [hunk 1]
```diff
@@ -56,7 +56,7 @@ struct MemoryInfo;
 
 namespace facebook::presto {
 
-// Three states our server can be in.
+// Three states server can be in.
```
Comment (xiaoxmeng) [inline presto-native-execution/presto_cpp/main/PrestoServer.h:59]:
```
/// for public comments in header file. Thanks!
```

**Review by xiaoxmeng — APPROVED** [review]
@amitkdutta LGTM. Thanks!

**Review by mbasmanova — COMMENTED** [review]
@amitkdutta "Minor refactor" is not descriptive. Perhaps, change to "Remove unnecessary includes from PrestoServer.cpp"

**Review by mbasmanova — APPROVED** [review]
Thanks.

**Review by majetideepak — APPROVED** [review]

**Pull request merged**